Assets/Packages/com.studio23.ss2.dialoguesystem/Runtime/Data/Models/CharacterTable.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Studio23.SS2.DialogueSystem.Data
+{
+    //todo remove this
+    [CreateAssetMenu(menuName = "Studio-23/Dialogue System/New Character Table")]
+    public class CharacterTable : ScriptableObject
+    {
+        [Serializable]
+        public class CharacterInfo
+        {
+            public string characterID;
+            public string characterName;
+            public List<CharacterReactionImage> Reactions;
+        }
+
+        [Serializable]
+        public class CharacterReactionImage
+        {
+            public string Reaction;
+            public Sprite Image;
+        }
```

Comment:
Let's keep every class under its separate script unless it is only used within that script and nowhere else.